pkg/proxy/userspace/proxier.go

```diff
@@ -260,8 +263,10 @@ func CleanupLeftovers(ipt iptables.Interface) (encounteredError bool) {
 
 // Sync is called to immediately synchronize the proxier state to iptables
 func (proxier *Proxier) Sync() {
-	if err := iptablesInit(proxier.iptables); err != nil {
-		glog.Errorf("Failed to ensure iptables: %v", err)
+	if runtime.GOOS != "windows" {
```

Comment:
Seems like these are details of the proxier implementation (or should be hidden under an interface method like .init()).